mozdownload/__init__.py

```diff
@@ -2,26 +2,31 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+__version__ = '1.19'
+
```

Comment:
Is there a reason you don't go ahead and do a version bump here? This is a pretty big change.